### PR TITLE
Fix typo x forwarded for

### PR DESCRIPTION
**Pull request by Danielecina**
- fix x forwarded for
- now host field is compliant with Mia-Platform logs

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- [BMP-507](https://makeitapp.atlassian.net/browse/BMP-507): fix x forwarded for
```
Comment (fredmaggiowski) [inline CHANGELOG.md:11]:
```suggestion
- [BMP-507](https://makeitapp.atlassian.net/browse/BMP-507): using `x-forwarded-for` header to get IP in logs
```

File: customwriter_test.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+					Message: "test",
+					Data:    logrus.Fields{},
+				}
+				c := JSONFormatter{}
+
+				result, err := c.Format(&logEntry)
+
+				assert.Assert(t, err == nil, "failed custom writer writing: %s", err)
+				assert.Equal(t, string(result), fmt.Sprintf("{\"level\":%d,\"msg\":\"test\",\"time\":%d}\n", testCase.expectedLevel, testCase.expectedTime))
+
```
Comment (fredmaggiowski) [inline customwriter_test.go:95]: Can you remove this empyt line?

File: inithelper_test.go [hunk 3]
```diff
@@ -46,4 +51,48 @@ func TestInitHelper(t *testing.T) {
...
 	})
+
+	t.Run("customWriter integration", func(t *testing.T) {
+		now := time.Now()
+		var buffer bytes.Buffer
+
+		logger, _ := InitHelper(InitOptions{})
+		logger.Out = &buffer
+		logger.WithTime(now)
+		logger.WithField("foo", "bar").Info("ciao")
```
Comment (fredmaggiowski) [inline inithelper_test.go:62]:
```suggestion
		logger.WithField("foo", "bar").Info("hello")
```

File: inithelper_test.go [hunk 4]
```diff
@@ -46,4 +51,48 @@ func TestInitHelper(t *testing.T) {
...
+			Time    int64  `json:"time"`
+		}
+
+		var result log
+		err := json.Unmarshal(buffer.Bytes(), &result)
+
+		assert.Assert(t, err == nil, "Unaxpected Error %s", err)
+		assert.DeepEqual(t, result, log{
+			Level:   int(30),
+			Message: "ciao",
```
Comment (fredmaggiowski) [inline inithelper_test.go:77]:
```suggestion
			Message: "hello",
```

File: inithelper_test.go [hunk 5]
```diff
@@ -46,4 +51,48 @@ func TestInitHelper(t *testing.T) {
...
+			Foo     string `json:"foo"`
+			Time    int64  `json:"time"`
+		}
+
+		var result log
+		err := json.Unmarshal(buffer.Bytes(), &result)
+
+		assert.Assert(t, err == nil, "Unaxpected Error %s", err)
+		assert.DeepEqual(t, result, log{
+			Level:   int(30),
```
Comment (fredmaggiowski) [inline inithelper_test.go:76]:
```suggestion
			Level:   30,
```

`int` conversion is redundant

File: inithelper_test.go [hunk 6]
```diff
@@ -46,4 +51,48 @@ func TestInitHelper(t *testing.T) {
...
+			Message: "ciao",
+			Foo:     "bar",
+			Time:    now.Unix(),
+		})
+	})
+}
+
+func captureStdout(t *testing.T, f func()) string {
```
Comment (fredmaggiowski) [inline inithelper_test.go:84]: This function is now useless, can you remove it?

Comment (Danielecina) [reply inithelper_test.go:84]: done

File: customwriter_test.go (hunk 2 above)
Comment (Danielecina) [reply customwriter_test.go:95]: done

File: inithelper_test.go [hunk 7]
```diff
@@ -46,4 +49,32 @@ func TestInitHelper(t *testing.T) {
...
+			Level   int    `json:"level"`
+			Message string `json:"msg"`
+			Foo     string `json:"foo"`
+			Time    int64  `json:"time"`
+		}
+
+		var result log
+		err := json.Unmarshal(buffer.Bytes(), &result)
+
+		assert.Assert(t, err == nil, "Unaxpected Error %s", err)
```
Comment (fredmaggiowski) [inline inithelper_test.go:72]:
```suggestion
		assert.Assert(t, err == nil, "Unexpected Error %s", err)
```

File: inithelper_test.go [hunk 8]
```diff
@@ -46,4 +49,32 @@ func TestInitHelper(t *testing.T) {
 		assert.Assert(t, logger == nil, "Logger is nil.")
 		assert.Assert(t, err != nil, "An error is expected. Found nil instead.")
 	})
+
+	t.Run("customWriter integration", func(t *testing.T) {
```
Comment (fredmaggiowski) [inline inithelper_test.go:53]:
```suggestion
	t.Run("custom JSONFormatter integration", func(t *testing.T) {
```

**Review by fredmaggiowski — APPROVED** [review]

File: logmiddleware_test.go [hunk 9]
```diff
@@ -135,6 +135,7 @@ func TestLogMiddleware(t *testing.T) {
 		str := buffer.String()
 
 		for i, value := range strings.Split(strings.TrimSpace(str), "\n") {
+			fmt.Printf("%+v\n", value)
```
Comment (fredmaggiowski) [inline logmiddleware_test.go:138]:
```suggestion
```

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**